test/functional/wallet_decommission_request.py

```diff
@@ -329,6 +320,11 @@ async def async_test(self):
             pools = await wallet.list_pool_ids()
             assert_equal(len(pools), 0)
 
-
 if __name__ == '__main__':
-    WalletSubmitTransaction().main()
+    tf1 = WalletDecommissionRequest()
+    tf1.use_wallet_to_produce_block = False
```

Comment:
Why not pass this in the constructor of that class? This looks very messy... Just add an `__init__()` function, and default that parameter there.